em-app/src/lib.rs

```diff
@@ -45,7 +46,28 @@ pub fn get_fortanix_em_certificate(url: &str, common_name: &str, signer: &mut dy
     get_certificate(url, common_name, signer, None, None)
 }
 
-pub fn get_certificate(url: &str, 
+pub fn get_certificate_subject(url: &str,
```

Comment:
We should try to keep our code well-formatted as much as possible. To achive that we can use:

```bash
cd em-app

cargo fmt
# or
rustfmt src/lib.rs
```

`cargo fmt` will format the entire crate
`rustfmt src/lib.rs` will format the entire file.

You should _not_ push these changes since there might be many formatting changes in code you did not change/add, but you can use the result of these commands as a reference to format your changes.

For instance, when I format this file I see that the `get_certificate_subject` is formatted as so:

```rust
pub fn get_certificate_subject(
    url: &str,
    subject: &Name,
    signer: &mut dyn CsrSigner,
    alt_names: Option<Vec<Cow<str>>>,
    config_id: Option<&str>,
) -> Result<FortanixEmCertificate> {
    let pub_key = signer.get_public_key_der()?;
    let user_data = get_user_data(&pub_key, config_id)?;

    let (attestation_certificate_der, node_certificate_der, csr_pem) =
        platform::get_remote_attestation_parameters_subject(
            signer, url, subject, &user_data, alt_names,
        )?;

    let certificate_response = request_issue_certificate(url, csr_pem)?;

    Ok(FortanixEmCertificate {
        attestation_certificate_der,
        node_certificate_der,
        certificate_response,
    })
}
```